src/components/MetricsPane/SelectedSiteMetrics.jsx

```diff
@@ -83,28 +89,100 @@ export const SelectedSiteMetrics = ({
     updateURLParams(queryParams)
   }
 
-  const handleSampleDateChange = ({ target: { value } }) => {
+  const handleSurveyChange = ({ target: { value } }) => {
     updateCurrentSampleEvent(value)
   }
 
+  const surveysAtSimilarSites = getSurverysAtSimilarSites({
+    projectData: displayedProjects,
+    surveyToCompareTo: selectedSampleEvent,
+  })
+
+  const getProjectSiteOptionLabel = (survey) => `${survey.project_name} - ${survey.site_name}`
+
+  // we only show one survey per site in the project-site drop down since the user will be able to select other surveys at each site from the sample date drop down
+  const surveysAtSimilarSitesLimitedToOneSurveyPerSite = surveysAtSimilarSites
```

Comment:
this name is super confusing... can it just be `surveyPerSite` ?

Reply:
The _concept_ is confusing! I struggled with it and may need your help with renaming. I want to communicate that this is surveys at similar sites to differentiate it from the _other_ drop down which lists surveys at the _same_ site. Further, in order for this to meet the requirements, we only show one survey per similar site (because the other surveys at that site will be listed in the other drop down).

Ill try `oneSurveyPerSimilarSiteList` which is easier to understand/skim I think

Reply:
The dropdown technically displays project/sites to the user in the label, but the value is a survey id if that helps make this more intuitive

Reply:
ahh ok that makes more sense. maybe `getSurveyPerSimilarSiteList` ?

Reply:
its not a getter, its the value.

Reply:
`oneSurveyPerSimilarSiteList` works !